torchvision/io/__init__.py

```diff
@@ -107,7 +111,7 @@ def __init__(self, path, stream="video"):
                 + "ffmpeg (version 4.2 is currently supported) and"
                 + "build torchvision from source."
             )
-        self._c = torch.classes.torchvision.Video(path, stream)
+        self._c = torch.classes.torchvision.Video(path, stream, num_threads)
```

Comment:
num_threads is not defined anywhere. Should it be an argument in `__init__()`?

Reply:
Ah yes - forgot to add this